autoload/pandoc/completion.vim

```diff
@@ -18,8 +18,28 @@ function! pandoc#completion#Init() abort "{{{1
             let g:pandoc#completion#bib#use_preview = 0
         endif
     endif
-
+    "
+    " If your images are in the same directory as the markdown file, they are included
```

Comment:
Is `figures` that standard? Most natural would be `images` (what I use). Also, rather `figs` than `fig` following the convention to name a directory in the plural form (`Documents`, `Pictures`, `Downloads`...)

Reply:
added a commit to take care of this.